### ISSUE TITLE
test directory not installed?

### ISSUE THREAD

**Issue by ppxasjsm** [michellab/Sire#86]
I have done a clean install on my mac. Everything seems to be working ok, apart form the tests:

```
./sire_test
"/Users/toni_brain/feat_sire.app/share/Sire/scripts/sire_test.py" 
Starting ./sire_test: number of threads equals 4
Traceback (most recent call last):
  File "/Users/toni_brain/feat_sire.app/share/Sire/scripts/sire_test.py", line 21, in <module>
    testdirs = os.listdir(testdir)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/toni_brain/feat_sire.app/test'
```

Seems that $HOME/sire.app/test is not installed.

**Comment by ppxasjsm** [comment]
Ok seems that they are, but in a different location than stored in Sire.Config.test_directory
Original location: $HOME/sire.app/test
Now: $HOME/sire.app/pkgs/sire-2015.0.0/test
This comes from the copying of sire to the package structure of anaconda I believe. and means that  sire_test fails because the Sire.Config.test_directory still points to the old location.

**Comment by chryswoods** [comment]
Yes. I am still undecided where the tests should be located so have not updated this variable. I don't want to clutter up the conda install, so am now leaning towards having the tests in pkgs/sire-2015.0.0/test. I think this may be cleaner, particularly if we let people layer in multiple Sire installations into a single conda (or want to make it easier to 'conda install sire' in the future)? What do you think?

**Comment by ppxasjsm** [comment]
Yes i agree. Where it is now is a good place. Where do you set the test_directory variable. I couldn't find it (didn't look very hard though)

**Comment by chryswoods** [comment]
It is set in wrapper/build/cmake/Config__init__.py.in

This file is processed by cmake to create the Sire/Config/__init__.py file that is imported when you import Sire.Config. At the moment it is set to

test_directory = "%s/test" % Sire.Base.getInstallDir()

but I need to (somehow) get hold of the pkgs/sire-2015.0.0 directory and use that instead...

**Comment by ppxasjsm** [comment]
Ah ok, that explains why I couldn't find it then! I thought it would be somewhere in the camke files.

**Comment by chryswoods** [comment]
I have a few of these "pre-processed" template cmake files. They are always in the directory build/cmake and are called FILE.in or FILE_template, e.g.

corelib/build/cmake/sire_config.h.in

or

corelib/build/cmake/sire_version_template.h

(I used .h.in when this was a file used to store Sire cmake config output, e.g. whether or not a header file existed, or number of bits on the machine, while I used _template.h when I had a file that was templated based on variables available at the time of running cmake, e.g. locations of install or library directories etc.)
